code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -15,6 +15,7 @@
 	var/icon_aggro = null // for swapping to when we get aggressive
 	see_in_dark = 8
 	see_invisible = SEE_INVISIBLE_MINIMUM
+	mob_size = MOB_SIZE_LARGE
```

Comment:
The goliath being its own mob size doesn't particularly have a purpose with the way they're currently implemented. Originally, goliaths were significantly more threatening than they are now, and as a result, gathering a bunch of goliath corpses in a locker and lazarus-ing them on the station was a huge threat, so someone made them only fit one to a locker. With them being significantly neutered in the threat department, them being the same mob size as everyone else should be fine.